src/components/listing/CategoryChips.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { getCategories } from "utils/service-requests/video-services";
+import styles from "./listing.module.css";
+
+const CategoryChips = ({ setCategory }) => {
+  const { categories } = getCategories();
+  const [isActive, setIsActive] = useState("");
+
+  return (
+    <div className="mx-xs my-s">
```

Comment:
you can extract this repeating button to a component

Reply:
Good suggestion. Noted.